### PR TITLE
Fetch cat

### PR DESCRIPTION
**Pull request by upluke**
### Pull Request Summary

Fetch all Categories from Amplify
Have frontend render the list of categories in the menu

### Pull Request Checklist

- [x] Code is neat, readable, and works
- [x] Comments are appropriate
- [x] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [x] The developer name is specified
- [x] The summary is completed
- [x] Ask for a review in communication channels

### Reviewer Checklist - PULL REQUEST REVIEWER ONLY

IMPORTANT: The rest of the sections in this checklist should only be filled out by authorized pull request reviewers. If you are the individual contributor, do not fill out the rest of the fields or check the boxes.

## Reviewer: Full Name

NOTE: Pull requests should only be merged when all boxes are checked.

- [ ] Assign yourself as reviewer if not assigned
- [ ] The reviewer name is specified
- [ ] The code is fully reviewed
- [ ] Meaningful feedback is given
- [ ] Let developer know a review has been made

### CODE REVIEW

**Comment by upluke** [comment]
Retained two buttons (Seed and Delete Seed) to test fetching all categories from Amplify and can be removed at any time
Screen Shot 2023-04-26 at 10 18 07 PM

File: src/components/Dropdown/DropdownStyles.css [hunk 1]
```diff
@@ -78,17 +85,17 @@
 .items {
   width: 116px;
   height: 24px;
-  font-weight: 400;
+  /* font-weight: 400;
```
Comment (jlaksana) [inline src/components/Dropdown/DropdownStyles.css:88]: Please remove any commented code

File: src/components/Dropdown/StyleDropdown.tsx [hunk 2]
```diff
@@ -15,16 +35,50 @@ export default function Styledropdown() {
...
       <button type="button" onClick={Showhide} className="dropbtn">
         <span id="Change"> {icon} </span>
         <span className="C1"> Style </span>
       </button>
+
       <div id="StyleDropdown" className="dropdown-content-three-items">
         <div className="font">
           <ul>
-            <li>
+            {/* <li>
```
Comment (jlaksana) [inline src/components/Dropdown/StyleDropdown.tsx:81]: same here. Remove all "dead" code

File: src/components/Dropdown/StyleDropdown.tsx [hunk 3]
```diff
@@ -15,16 +35,50 @@ export default function Styledropdown() {
...
+
+  // this functin can be removed any timee, just for testing purposes
+  const handleDeleteSeed = async () => {
+    await DataStore.delete(Category, (category) =>
+      category.name.contains("test")
+    ); // delete all the posts
+  };
   return (
     <div className="dropdown">
+      {/* div and two buttons two buttons can be removed at any time, just for testing purposes */}
```
Comment (jlaksana) [inline src/components/Dropdown/StyleDropdown.tsx:58]: Thanks for doing this. Made testing much easier. Before merging, please delete the seeding functionality

**Review by jlaksana — APPROVED** [review]
Looks good. Thanks for making the changes!

**Pull request merged**